features/unhandled_controller.feature

```diff
@@ -3,9 +3,9 @@ Feature: Unhandled exceptions in controllers support
 Scenario: Unhandled exceptions are delivered from controllers
   Given I set environment variable "BUGSNAG_API_KEY" to "a35a2a72bd230ac0aa0f52715bbdc6aa"
   And I configure the bugsnag endpoint
-  And I start the service "laravel"
-  And I wait for the app to respond on port "61280"
-  When I navigate to the route "/unhandled_controller_exception" on port "61280"
+  And I start the laravel fixture
+  And I wait for the app to respond on the appropriate port
```

Comment:
I added `the appropriate port` to indicate that we're acquiring the correct port here, but left it off the navigation step below.  I could go either way but it should probably be consistent.